package.json

```diff
@@ -187,7 +189,7 @@
         "ngc": "ngc -p tsconfig.app.json",
         "cleanup": "rimraf build/resources/main/static/ build/resources/main/aot",
         "clean-www": "rimraf build/resources/main/static/app/{src,build/}",
-        "start": "yarn run webpack:dev",
+        "start": "NODE_OPTIONS=--max_old_space_size=4096 yarn run webpack:dev",
```

Comment:
This environmental variable breaks the windows shell (and powershell).
<img width="486" alt="CommandFailure" src="https://user-images.githubusercontent.com/12861668/119849784-346d7200-bf0d-11eb-908c-54d6cb4604d0.PNG">

I suggest moving the "NODE_OPTIONS=--max_old_space_size=4096" part down into the webpack-ts command since it already uses cross-env:
```json
{
    "start": "yarn run webpack:dev",
    "webpack:dev": "yarn run webpack-ts serve --config webpack/webpack.dev.ts --inline --hot --port=9060 --watch-content-base",
    "webpack-ts": "cross-env NODE_OPTIONS=--max_old_space_size=4096 TS_NODE_PROJECT=\"tsconfig.webpack.json\" webpack"
}
```

Reply:
Thanks for pointing that out! I'll fix this.

Reply:
I think that's alright. Thank you!